Makefile

```diff
@@ -53,5 +53,12 @@ tests/GNTAllocation.bin: contracts/GNTAllocation.sol
 tests/GNTAllocation.abi: contracts/GNTAllocation.sol
 	solc --bin --abi --optimize contracts/GNTAllocation.sol | awk '/======= GNTAllocation =======/,/======= GolemNetworkToken =======/' | grep '\[.*\]' > tests/GNTAllocation.abi
 
+tests/Wallet.bin: contracts/Wallet.sol
```

Comment:
Yes, that's good idea, but: for verification we cannot use imports (we might prepare single file code just for verification) and it would be good to distinguish contracts for tests only from GNT crowdfunding ones.